src/lib/SapphireClient.ts

```diff
@@ -108,6 +108,12 @@ export interface SapphireClientOptions {
 	 * @default true
 	 */
 	loadDefaultErrorListeners?: boolean;
+
+	/**
+	 * Display "bot is typing..." when a command is accepted.
```

Comment:
```suggestion
	 * Controls whether the bot will automatically appear to be typing when a command is accepted.
```

Reply:
"Controls whether the bot will automatically appear to be" feels very awkward. How about "Whether or not the bot should appear to be typing when a command is accepted."?